common/src/main/java/org/apache/celeborn/common/protocol/PartitionLocation.java

```diff
@@ -286,13 +285,9 @@ public String toString() {
       peerAddr = peer.hostAndPorts();
     }
     return "PartitionLocation["
-        + "\n  id(rawId-attemptId)-epoch:"
+        + "\n  id-epoch:"
```

Comment:
if we want to look at shufflewring filename of one map task, it's not directly to build the relation between filename (  id + "-" + epoch + "-" + mode.mode) and the mapid

Reply:
It's partial right, as the `<int> mapId `also comes from `LifecycleManager`. Only `LifecycleManager` has the knowledge which partition belongs to the actual resultPartition.